### PR TITLE
Bigquerystorage: fix missing dependency.

### LINKED ISSUE #6710: com.google.cloud:google-cloud-bigquerystorage is missing from the MOB
The BigQuery Storage client com.google.cloud:google-cloud-bigquerystorage is missing from the published BOM.

Notice that the two underlining libraries are in the BOM:
* com.google.api.grpc:proto-google-cloud-bigquerystorage-v1beta1
* com.google.api.grpc:grpc-google-cloud-bigquerystorage-v1beta1

### PR DESCRIPTION
**Pull request by pmakani**
Fixes #6710

### CODE REVIEW

File: google-cloud-bom/pom.xml [hunk 1]
```diff
@@ -258,6 +258,11 @@
         <artifactId>grpc-google-cloud-bigquerydatatransfer-v1</artifactId>
         <version>0.83.1-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-bigquerydatatransfer-v1:current} -->
       </dependency>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-bigquerystorage</artifactId>
+        <version>0.118.1-beta-SNAPSHOT</version><!-- {x-version-update:google-cloud-bigquerystorage:current} -->
```
Comment (elharo) [inline google-cloud-bom/pom.xml]: This shouldn't be a snapshot version and maybe not 0.118.1

Comment (pmakani) [reply google-cloud-bom/pom.xml]: Done.

**Comment by pmakani** [comment]
@davidrabinowitz PTAL

**Comment by davidrabinowitz** [comment]
Great, thanks!

**Comment by pmakani** [comment]
@elharo PTAL.

**Pull request merged**